composer.json

```diff
@@ -22,8 +22,7 @@
     "require": {
         "php": "^8.3",
         "silverstripe/admin": "^3",
-        "silverstripe/framework": "^6",
-        "silverstripe/versioned": "^3"
```

Comment:
Why is the dependency on versioned being removed??

Reply:
Accident I think, added back in